core/src/main/scala/cats/data/package.scala

```diff
@@ -77,6 +77,11 @@ package object data extends ScalaVersionSpecificPackage {
   type RWS[E, L, S, A] = ReaderWriterState[E, L, S, A]
   val RWS = ReaderWriterState
 
+  type Env[E, A] = EnvT[Id, E, A]
+  object Env {
+    def apply[E, A](e: E, a: Id[A]): Env[E, A] = EnvT(a, e)
```

Comment:
Any reason not just `a: A` here?

The arguments are inconsistent with the order in `EnvT.apply`.

Reply:
Well, the compiler complained, but a type annotation worked.